### PR TITLE
Convert Schnorr tests to typescript

### PR DESCRIPTION
**Pull request by HenryNguyen5**

### CODE REVIEW

File: evm/v0.5/testv2/Schnorr/fixtures/dssTest.ts [hunk 1]
```diff
@@ -1,11 +1,9 @@
 // This test was generated by by kyber/sign/ethdss/ethdss_test.go, with
 // printTests set to true.
-const dssTest = [
+export const dssTest = [
```
Comment (se3000) [inline evm/v0.5/testv2/Schnorr/fixtures/dssTest.ts]: What do you think of just adding this into the file it's consumed in? It seems to only be used in one place.

File: evm/v0.5/testv2/Schnorr/fixtures/tests.ts [hunk 2]
```diff
@@ -1,6 +1,6 @@
 // These tests were generated by by kyber/sign/ethschnorr/ethschnorr_test.go,
 // with printTests set to true.
-const tests = [
+export const tests = [
```
Comment (se3000) [inline evm/v0.5/testv2/Schnorr/fixtures/tests.ts]: Same with this. Seems significantly simpler to just include it in `Schnorr.test.ts`.

Comment (HenryNguyen5) [reply evm/v0.5/testv2/Schnorr/fixtures/tests.ts]: I wouldn't mind adding them to the same fixture file, i don't think fixtures this large should belong within the same file as the tests though

**Comment by HenryNguyen5** [comment]
@se3000 ill do a second pass refactor later on

**Pull request merged**